src/Tests/ObjectLifetimeTests/Directory.Build.targets

```diff
@@ -1,2 +1,5 @@
 <Project>
+    <Import Project="CSWinRTInApp.targets"/>
+    <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />    
+    <Import Condition="'$(MSBuildProjectExtension)' == '.csproj' and '$(SimulateCsWinRTNugetReference)' == 'true'" Project="..\..\..\nuget\Microsoft.Windows.CsWinRT.targets" />
```

Comment:
Would the same apply here that this comes from the import?

Reply:
Yes, removed.